designer/server/src/test/scala/pl/touk/nussknacker/ui/validation/CustomActionValidatorTest.scala

```diff
@@ -0,0 +1,198 @@
+package pl.touk.nussknacker.ui.validation
+
+import cats.data.{Validated, ValidatedNel}
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.engine.api.context.PartSubGraphCompilationError
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.{EmptyMandatoryParameter, MismatchParameter}
+import pl.touk.nussknacker.engine.api.definition.{
+  MandatoryParameterValidator,
+  NotBlankParameterValidator,
+  NotNullParameterValidator,
+  StringParameterEditor
+}
+import pl.touk.nussknacker.engine.api.deployment.ScenarioActionName
+import pl.touk.nussknacker.engine.api.parameter.ParameterName
+import pl.touk.nussknacker.engine.deployment.{CustomActionDefinition, CustomActionParameter}
+import pl.touk.nussknacker.restmodel.CustomActionRequest
+
+class CustomActionValidatorTest extends AnyFunSuite with Matchers {
+
+  private val testCustomActionName = ScenarioActionName("testCustomAction")
+
+  private val testCustomActionParams =
+    CustomActionParameter(
+      "testParam1",
+      StringParameterEditor,
+      NotNullParameterValidator ::
+        Nil
+    ) ::
+      CustomActionParameter(
+        "testParam2",
+        StringParameterEditor,
+        NotBlankParameterValidator :: NotNullParameterValidator :: Nil
+      ) :: Nil
+
+  private val testCustomAction =
+    CustomActionDefinition(
+      testCustomActionName,
+      "testStatus1" :: "testStatus2" :: Nil,
+      testCustomActionParams
+    )
+
+  private val noParamsCustomAction =
+    CustomActionDefinition(ScenarioActionName("noparams"), "testStatus" :: Nil, Nil)
+
+  private val validator = new CustomActionValidator(testCustomAction)
+
+  type TestParams = Map[String, String]
```

Comment:
It it used only in one place. Seems to be obsolete.

Reply:
removed